install/pre-installation.md

```diff
@@ -164,6 +164,11 @@ On first login, the LiveCD will prompt the administrator to change the password.
       site_dns=
       ```
 
+      ```bash
+      # Name of the system. This will only be used for the pit hostname.
+      system_name=
```

Comment:
This should be capitalized to align with 1.5 Set reusable environment variables. This step can be redundant to that section, because that section is also a reference section for folks to jump to and set consistent reusable vars.